### ISSUE TITLE
164 undeleted objects left

### ISSUE THREAD

**Issue by kenorb** [EA31337/EA31337#162]
Warnings while backtesting in CI:
```
3	15:10:15	2018.01.30 23:59  EA31337-Lite-Release-v1.078 EURUSD,M30: 164 undeleted objects left
3	15:10:15	2018.01.30 23:59  EA31337-Lite-Release-v1.078 EURUSD,M30: 82 objects of type Order left
3	15:10:15	2018.01.30 23:59  EA31337-Lite-Release-v1.078 EURUSD,M30: 82 objects of type Log left
3	15:10:15	2018.01.30 23:59  EA31337-Lite-Release-v1.078 EURUSD,M30: 82 leaked strings left
3	15:10:15	2018.01.30 23:59  EA31337-Lite-Release-v1.078 EURUSD,M30: 50848 bytes of leaked memory
```

- Branch: `master`
- Build: [575382111](https://travis-ci.org/EA31337/EA31337/builds/575382111)

This is regression, as build [574611265](https://travis-ci.org/EA31337/EA31337/jobs/574611265) got the warnings, where the previous build [574284544](https://travis-ci.org/EA31337/EA31337/jobs/574284544) doesn't have.

**Comment by kenorb** [comment]
Fixes: 21646c2cae398f068413358e51237bd9784be7a1 & 8e2629417184ac3522d0ca736dec89699d325d74

**Comment by kenorb** [comment]
Fixed in GH-160
